appservice/src/deploy/waitForDeploymentToComplete.ts

```diff
@@ -191,3 +191,11 @@ async function tryGetLatestDeployment(context: IActionContext, kuduClient: KuduC
 
     return [deployment, permanentId, initialStartTime];
 }
+
+function addAttemptTelemetry(context: IActionContext, methodName: string, attempt: number): void {
+    const key: string = methodName + 'MaxAttempt';
+    const existingValue: number | undefined = context.telemetry.measurements[key];
```

Comment:
This is probably fine as it is but I was a bit confused reading it. Maybe this would help?

```suggestion
    const existingAttempt: number | undefined = context.telemetry.measurements[key];
```